config/example/config.txt

```diff
@@ -208,6 +208,9 @@ GUEST_BAN
 ## Ban appeals URL - usually for a forum or wherever people should go to contact your admins.
 # BANAPPEALS http://example.com
 
+## Discord URL - Used for the "Join the Discord" verb in OOC.

```

Comment:
@WestHybrid You'll need to pay special mind to this, for when you merge. You'll have to add this to the existing config.